src/server/req_manager.c

```diff
@@ -1747,6 +1747,16 @@ mgr_server_unset(struct batch_request *preq)
 			} else if (strcasecmp(plist->al_name, ATTR_scheduling) == 0)
 				set_attr_svr(&(server.sv_attr[(int)SRV_ATR_scheduling]),
 					    &svr_attr_def[(int) SRV_ATR_scheduling], "TRUE");
+			else if (strcasecmp(plist->al_name, ATTR_job_sort_formula) == 0) {
+				pbs_sched *psched;
+
+				/* Ask all scheds to reconfigure */
+				for (psched = (pbs_sched *) GET_NEXT(svr_allscheds);
+								psched != NULL;
+								psched = (pbs_sched *) GET_NEXT(psched->sc_link)) {
+					set_scheduler_flag(SCH_CONFIGURE, psched);
```

Comment:
Why a SCH_CONFIGURE?  That just forces the scheduler to read sched_config.  There is nothing in there about the formula.  If you want to follow some of the other sched attrs, send a SCH_ATTRS_CONFIGURE.

Reply:
Actually, we don't need this at all since it's being read every sched cycle.

SCH_ATTRS_CONFIGURE would work if we were explicitly setting the sched object's job_sort_formula when the server's job_sort_formula was set. But we can't do that if we want to make server & sched's job_sort_formula attributes incompatible, right? e.g - if we set server's job_sort_formula to "1*walltime" and also set it on all sched objects inside validate_job_formula(), then later if we change the server's job_sort_formula to "2*walltime", it becomes difficult to tell if the previously set job_sort_formula value on sched objects was set via server's job_sort_formula, or if they were set explicitly on the sched objects. If they were set explicitly, then they would be incompatible, if they were set via the server's job_sort_formula, then it's okay to change them. There's no way to tell.